common/src/sql/dbinit.sql

```diff
@@ -912,6 +912,14 @@ CREATE TABLE omicron.public.role_assignment_builtin (
     PRIMARY KEY(user_builtin_id, resource_type, resource_id, role_name)
 );
 
+/*
+ * Store addresses allocated for internal services (like propolis zones)
+ */
+CREATE TABLE omicron.public.static_v6_address (
+    address STRING(64) PRIMARY KEY NOT NULL,
```

Comment:
I imagine we'd want to use the `INET` type here?

Reply:
Done, thanks - didn't know about this type :)